### ISSUE TITLE
Pulling correct Magic Type / Redwood.js Integration

### ISSUE THREAD

**Issue by Burnsy** [MagicHQ/magic-js#91]
### ✅ Prerequisites

- [x] Did you perform a cursory search of open issues? Is this question already asked elsewhere?
- [x] Are you reporting to the correct repository (`magic-sdk`)?

### ❓ Question

Best Method to pull types for Magic Client (New ...

I have been working on the auth implementation of magic links for Redwood.js! 

We have most of the implementation done but i could not find an easy way to pull the client type into the auth wrapper. 

is there a simpler type than  
```
SDKBase<MagicSDKAdditionalConfiguration<string, any>> & {
    [x: string]: Pick<Extension<string>, never>;
}
```

if you wanted to quickly look through the pull request and check that i have implemented it all properly please do! I would love some insight. Pull request - [here](https://github.com/redwoodjs/redwood/pull/562)

Thanks

**Comment by smithki** [comment]
Hey, thanks for opening an issue. You're right, we need to export a type alias for the constructor (we wrap our base constructors with some pretty complex typings to enable our extension architecture). I've left a comment on your PR with an interim solution, let me know how that works!

**Comment by smithki** [comment]
I have a proposed fix in #92. I'll post another update once it passes testing and releases.

**Comment by smithki** [comment]
Typings fix landed in `v1.3.4`. Please re-open this issue if you notice an lingering bugs!

**Comment by smithki** [comment]
:rocket: Issue was released in [`@magic-sdk/provider@3.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/provider%403.0.1), [`@magic-sdk/react-native@3.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/react-native%403.0.1), [`@magic-sdk/types@2.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/types%402.0.1), [`magic-sdk@3.0.1`](https://github.com/magiclabs/magic-js/releases/tag/magic-sdk%403.0.1) :rocket:

**Comment by smithki** [comment]
:rocket: Issue was released in [`@magic-sdk/provider@3.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/provider%403.0.1), [`@magic-sdk/react-native@3.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/react-native%403.0.1), [`@magic-sdk/types@2.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/types%402.0.1), [`magic-sdk@3.0.1`](https://github.com/magiclabs/magic-js/releases/tag/magic-sdk%403.0.1) :rocket:
